chatgpt-shell-google.el

```diff
@@ -94,7 +94,12 @@ VALIDATE-COMMAND handler."
                                          :short-version "1.5-flash"
                                          :path "/v1beta/models/gemini-1.5-flash"
                                          :token-width 4
-                                         :context-window 1048576)))
+                                         :context-window 1048576)
+        (chatgpt-shell-google-make-model :version "gemini-2.0-flash-exp"
+                                         :short-version "2.0-flash-exp"
+                                         :path "/v1beta/models/gemini-2.0-flash-exp"
+                                         :token-width 4
+                                         :context-window 2097152)))
```

Comment:
The context window of `gemini-2.0-flash-exp` is `1048576`?

![CleanShot 2024-12-20 at 19 12 44@2x](https://github.com/user-attachments/assets/3542186c-965c-417a-b2d3-9c657515debe)

Reply:
Good catch, I just changed it.

Reply:
Thanks!